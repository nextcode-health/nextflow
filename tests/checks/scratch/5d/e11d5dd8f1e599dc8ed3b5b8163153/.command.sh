#!/bin/bash -ue
ls dir1 | sort
mkdir dir2
echo hello > dir2/alpha.txt
echo world > dir2/beta.txt
