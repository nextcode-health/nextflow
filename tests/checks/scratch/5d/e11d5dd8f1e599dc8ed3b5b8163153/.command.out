link_1.fasta
link_2.fasta
link_3.fasta
